controllers/front/ContactController.php

```diff
@@ -28,6 +28,14 @@ class ContactControllerCore extends FrontController
 {
     public $php_self = 'contact';
     public $ssl = true;
+    public $auth = true;
+    
+    public function __construct()
+    {
+        parent::__construct();
+        
+        $this->authRedirection = $this->context->link->getPageLink($this->php_self);
+    }
```

Comment:
I guess that change wasn't wanted in this PR?

Reply:
I have thus defended some of the sites from bots that spread spam

Reply:
I deleted this fragment